src/components/CustomModal/index.tsx

```diff
@@ -1,16 +1,65 @@
 import React from "react";
-import { Modal, ModalProps } from "antd";
+import { Divider, Modal, ModalProps } from "antd";
 import classNames from "classnames";
 
 import theme from "../theme/light-theme.css";
 import styles from "./style.css";
 
-/** Tiny wrapper component to keep modal styling consistent */
-const CustomModal: React.FC<ModalProps> = (props) => (
-    <Modal
-        {...props}
-        className={classNames(theme.lightTheme, styles.modal, props.className)}
-    />
-);
+interface CustomModalProps extends ModalProps {
+    closeHandler: () => void;
```

Comment:
110% optional suggestion because I'm a weird type-soundness obsessive: could remove the `Modal` props that are always overwritten from `CustomModalProps`:

```tsx
type OmittedProps =  "onCancel" | "closable" | "title" | "footer" | "open" | "centered";
interface CustomModalProps extends Omit<ModalProps, OmittedProps> {
  // ...
}
```

Reply:
Interesting I've never really looked closely at this `Omit` keyword. I aspire to be as type-sound obsessive, although I've lived my life as a more ham-fisted typist.

Going to add this change! But repeating the justification mostly for my own understanding: without the `Omit` its possible that `CustomModals` could get passed `Modal` props we aren't expecting and then they would behave in unexpected ways, so its better to only allow the ones we've explicitly accounted for.